src/prog_models/utils/calc_error.py

```diff
@@ -7,6 +7,7 @@
 
 from collections.abc import Iterable
 from typing import List
+from fastdtw import *
```

Comment:
Also, let's move this into the dtw function since it's only needed if you use that function (which most wont). 

I *think* python will only import it once (because it keeps a list in sys.modules), but we should confirm